cli/tests/059_fs_relative_path_perm.ts.out

```diff
@@ -1,4 +1,4 @@
-[WILDCARD]error: Uncaught PermissionDenied: read access to "non-existent", run again with the --allow-read flag
+[WILDCARD]error: Uncaught PermissionDenied: Access to read "non-existent" required, run again with read permission
```

Comment:
The message should contain the flag suggestion. The wording is a bit off for the global case as well. I sense that the capitalisation of the permission name is a problem? How about just:
```suggestion
[WILDCARD]error: Uncaught PermissionDenied: Requires read access to "non-existent", run again with --allow-read.
```

Reply:
I agree with Nayeem, the message should stay as is and ideally we could conditionally skip the part with flag to address #9002

Reply:
@bartlomieju is it better now?
`Access to read "non-existent" required, run again with the --allow-read flag`

Reply:
IMO they are still less concise/uniform than the original messages.
- `Access to read "foo" required`
- `Access to write to "foo" required` 🤮
- `Access to network for "foo:80" required`
- `Access to run a subprocess required`

vs original:
- `read access to "foo" required`
- `write access to "foo" required`
- `network access to "foo:80" required`
- `access to run a subprocess required`

My suggestion:
- `Requires read access to "foo"`
- `Requires write access to "foo"`
- `Requires network access to "foo:80"`
- `Requires subprocess access`

Reply:
I agree with Nayeem

Reply:
`Requires read access to "foo"` sounds a bit off to me, as there is no subject for `Requires`. I know `deno` is implied, but it still sounds weird

Reply:
> `Requires read access to "foo"` sounds a bit off to me, as there is no subject for `Requires`. I know `deno` is implied, but it still sounds weird

The implied subject is the operation that failed. I think plenty of error messages are structured like that.

Reply:
I forgot for a moment that error message in JS would be `PermissionDenied: Requires read access to "foo", run again with --allow-read.`, so it would actually work. will rework the messages.

Reply:
Alright, error messages now are 
```
Requires net access to "google.com", run again with the --allow-net flag
Requires read access to "./test.ts", run again with the --allow-read flag
Requires write access to "./test.ts", run again with the --allow-write flag
Requires env access, run again with the --allow-env flag
Requires run access, run again with the --allow-run flag
Requires plugin access, run again with the --allow-plugin flag
```
and debug log messages are
```
Granted net access to "www.google.com"
Granted read access to "./test.ts"
Granted write access to "./test.ts"
Granted env access
Granted run access
Granted plugin access
```
will this do?